server/services/DestinationsService.js

```diff
@@ -175,17 +175,24 @@ export default class DestinationsService extends MDSEnabledClientService {
         },
       });
     } catch (err) {
+      // Indices will be created when the monitor is created.
       if (isIndexNotFoundError(err)) {
         return res.ok({
-          body: { ok: false, resp: {} },
+          body: {
+            ok: false,
+            totalMonitors: 0, 
+            monitors: [],
+            message: "Config index will be created automatically when the monitor is created"
```

Comment:
I would rather keep the schema same and return a known error message which we can check for in the frontend instead of introducing a new field `totalMonitors`